pkg/server/container_create_windows.go

```diff
@@ -279,9 +280,22 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 
 	// Add OCI Mounts
 	for _, m := range config.GetMounts() {
+
+		//normalize the format of the container path
+		var formattedDestination string
```

Comment:
Do we need to do this for `m.HostPath` as well? Seems like Kube could also pass `c:/test/test` as a path for the `HostPath`?

Reply:
I think kubelet handles host path correctly, but better safe than sorry I guess.